MetaMask/metamask-extension
# Use core signature controller

## Explanation

Replace the `SignController` with the core equivalent from the `@metamask/signature-controller` package.

## Manual Testing Steps

1. Go to test dApp.
2. Verify `Eth Sign`.
3. Verify `Personal Sign`.
4. Verify all versions of `Sign Typed Data`.
5. Verify `Sign In With Ethereum`.

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
